src/groupeddataframe/splitapplycombine.jl

```diff
@@ -766,49 +799,18 @@ function _combine(f::AbstractVector{<:Pair}, gd::GroupedDataFrame)
     if !all(x -> length(x) == length(outcols[1]), outcols)
         throw(ArgumentError("all functions must return values of the same length"))
     end
-    nams = [f[i] isa Pair{<:ColumnIndex} ?
-                Symbol(names(gd.parent)[index(gd.parent)[first(f[i])]],
-                       '_', funname(last(f[i]))) :
-                Symbol('x', i)
-            for i in 1:length(f)]
-    valscat = DataFrame(collect(AbstractVector, outcols), nams, makeunique=true)
-    return idx, valscat
+    return idx, DataFrame(collect(AbstractVector, outcols), nms)
```

Comment:
This could still return a vector instead of a `DataFrame`?

Reply:
No, because the other `_combine` (taking a `Function`) must return a `DataFrame` as the names are there generated dynamically (and later we immediately need a `DataFrame` anyway).